Tests/PointFreeTests/EmailTests/__Snapshots__/TeamEmailsTests/testTeammateRemovedEmailView.1.html

```diff
@@ -106,7 +106,9 @@
                             margin-left:0;
                             padding-top:1rem;
                             padding-bottom:1rem">
-                    You have removed Blob from your Point-Free team, which means they no longer
+                    You have removed 
+                    Blob
+                     from your Point-Free team, which means they no longer
```

Comment:
I don't know why the text is breaking this way now, but it doesn't affect the screenshot so I think it's ok.